fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/operations/NonRdfSourceOperation.java

```diff
@@ -63,5 +63,5 @@
     /**
      * @return The size in bytes of content associated with this resource.
      */
-    long getContentSize();
+    Long getContentSize();
```

Comment:
Just curious why you changed this from a primitive to a Long object.

Reply:
so that null sizes can be passed in, in the case that the size is not known/specified, rather than recording 0